src/main/java/org/micks/champmaker/auth/AuthController.java

```diff
@@ -7,7 +7,7 @@
 
 @Slf4j
 @RestController
-@RequestMapping("/auth")
+@RequestMapping("/auth/public")
```

Comment:
`public` should be first, so `/public/auth`. Other way it might not be recognized as public endpoint by API Gateway.

Reply:
Fixed